code/game/objects/items.dm

```diff
@@ -1351,6 +1351,7 @@
 	if((item_flags & ABSTRACT) || HAS_TRAIT(src, TRAIT_NODROP))
 		return
 	user.dropItemToGround(src, silent = TRUE)
+	animate(flags = ANIMATION_END_NOW) // clears the drop animation
```

Comment:
If this pr is not accepted I recommend doing this change in a separate pr cause it seems good